os/include/sys/sock_internal.h

```diff
@@ -59,6 +59,7 @@
 
 #include <tinyara/config.h>
 #include <sys/types.h>
+#include <uio.h>
 
```

Comment:
move header <uio.h> to sys/socket.h

Reply:
Modification completed. Thanks for the comment.